src/python/pants/backend/python/providers/python_build_standalone/rules.py

```diff
@@ -40,6 +42,8 @@
 PBS_NAMED_CACHE_NAME = "python_build_standalone"
 PBS_APPEND_ONLY_CACHES = FrozenDict({PBS_NAMED_CACHE_NAME: PBS_SANDBOX_NAME})
 
+StrPath = Union[str, bytes, os.PathLike]
```

Comment:
This just replicates the `_typeshed` types to match the `shutil.copy` signature, o/w can just narrow this down to `str` if there's a strong preference one way or another